src/v2/api/index.md

```diff
@@ -2197,6 +2197,11 @@ type: api
   ```
 
   `.camel` is not needed if you are using string templates, or compiling with `vue-loader`/`vueify`.
+  
+- **Note on truthiness and falsiness**
+  When used with `v-bind`, the only falsy values are now: `null`, `undefined`, and `false`. This means `0` and empty strings will render as truthy. So for example, `v-bind:draggable="''"` will render as `draggable="true"`.
+  
+  For enumerated attributes, in addition to the falsy values above, the string `"false"` will also render as `attr="false"`.
```

Comment:
```suggestion
  For enumerated attributes, in addition to the falsy values above, the string `'false'` will also be rendered as `attr="false"`.
```

Reply:
Can we have an example to make this statement clearer? I'm not sure a beginner will get what "enumerated attributes" are.

Reply:
added